### ISSUE TITLE
Cycle dependency breaks librarian

### ISSUE THREAD

**Issue by Soulou** [higanworks-cookbooks/hwraid#5]
I'm running the last version of librarian and when I want to add hwraid cookbook, I've the following error. I suppose it is because of cycle depdendency.

https://github.com/higanworks-cookbooks/hwraid/blob/master/metadata.rb#L18

What do you think?

```
[Librarian]     Checking hwraid/0.3.1 <https://github.com/higanworks-cookbooks/hwraid#master>
[Librarian]       Cycle with hwraid/0.3.1 <https://github.com/higanworks-cookbooks/hwraid#master>
[Librarian]       Backtracking from hwraid/0.3.1 <https://github.com/higanworks-cookbooks/hwraid#master>
[Librarian]   Failed to resolve hwraid (>= 0) <https://github.com/higanworks-cookbooks/hwraid#master>
Could not resolve the dependencies.
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/librarian-0.1.2/lib/librarian/action/persist_resolution_mixin.rb:11:in `persist_resolution'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/librarian-0.1.2/lib/librarian/action/update.rb:25:in `run'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/librarian-0.1.2/lib/librarian/cli.rb:173:in `update!'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/librarian-0.1.2/lib/librarian/cli.rb:115:in `update'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/librarian-0.1.2/lib/librarian/cli.rb:26:in `block (2 levels) in bin!'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/librarian-0.1.2/lib/librarian/cli.rb:31:in `returning_status'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/librarian-0.1.2/lib/librarian/cli.rb:26:in `block in bin!'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/librarian-0.1.2/lib/librarian/cli.rb:47:in `with_environment'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/librarian-0.1.2/lib/librarian/cli.rb:26:in `bin!'
/home/leo/.rbenv/versions/2.1.2/lib/ruby/gems/2.1.0/gems/librarian-chef-0.0.4/bin/librarian-chef:7:in `<top (required)>'
/home/leo/.rbenv/versions/2.1.2/bin/librarian-chef:23:in `load'
/home/leo/.rbenv/versions/2.1.2/bin/librarian-chef:23:in `<main>'
```

**Comment by Soulou** [comment]
I've forked the cookbook and removed the line:

https://github.com/Appsdeck/hwraid/blob/master/metadata.rb

I confirm this line is causing this dependency error with librarian, is it really necessary for foodcritic?

**Comment by sawanoboly** [comment]
@Soulou Thanks for your report.

I'll remove that line and release new version.

**Comment by sawanoboly** [comment]
I've released v0.3.2 https://supermarket.getchef.com/cookbooks/hwraid
